tests/contrib/hooks/wasb_hook.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
```

Comment:
Your tests are not executed on Travis. You will need to include them in "init.py" or rename the file to test_XXX.py